packages/figma-plugins/squid/src/ui/App.tsx

```diff
@@ -41,29 +42,21 @@ function App() {
         setSelectedObjects(data)
       }
       if (action === 'COPY_RESPONSE') {
-        if (typeof data !== 'string') return
-        const copyPromise = new Promise((resolve) => {
-          document.addEventListener('copy', async (event) => {
-            if (!event.clipboardData) return resolve(false)
-            event.preventDefault()
-            event.clipboardData.setData('text/plain', data)
-            await sleep(1000) // need to sleep a while to make sure the clipboard is updated before closing
-            resolve(true)
-          })
-        })
-        const textarea = document.createElement('textarea')
-        textarea.setAttribute('hidden', 'true')
-        textarea.value = 'Copying failed, try again in desktop app'
-        document.body.appendChild(textarea)
-        textarea.focus()
-        textarea.select()
-        document.execCommand('copy')
-        const copyResult = await copyPromise
+        if (typeof data !== 'object') return
```

Comment:
What if `data` equals `null`? Or it's not possible?

Reply:
thx, pushed new version